README.md

```diff
@@ -67,8 +67,10 @@ This connects a client that pings the `HelloWorker` and `DiceWorker` every few s
 ## Logs location
 The logs are stored in `logs` subdirectory of the location where your `spatialos.json` file is. For example, if my `spatialos.json` file is in `/workspace/project/SpatialOS`, the SpatialOS logs will go to `/workspace/project/SpatialOS/logs`.
 
-## Reference documentation
+## Known Issues
+* Our [Launcher](https://docs.improbable.io/reference/latest/shared/operate/launcher#the-launcher) currently only supports Unity and Unreal based [client workers](docs/reference/client-worker-configuration.md). As a result, you will not be able to start the client worker provided in this repository from the Launcher, if you deploy this project in the cloud.
```

Comment:
```suggestion
* Our [Launcher](https://docs.improbable.io/reference/latest/shared/operate/launcher#the-launcher) currently only supports Unity and Unreal based [client-workers](docs/reference/client-worker-configuration.md). As a result, if you deploy this project to the cloud, you can't start the provided client-worker from the Launcher.
```

Reply:
Is `client-worker` (with the `-` in the middle) a convention?

Reply:
Yes, server-worker and client-worker are conventions we (tech writers) are trying to push for throughout the docs.

You might also have noticed my adding "type" or "instance" in a couple of places - this is another thing we hope to make consistent, but it's a bit harder because it involves going through every single mention of "worker" and deciding what exactly we're talking about... so for the moment we have to live with some inconsistency.

Reply:
Applied your suggestion.